arangod/Aql/Functions.cpp

```diff
@@ -1604,6 +1604,74 @@ AqlValue Functions::Sleep(arangodb::aql::Query* query,
   return AqlValue(AqlValueHintNull());
 }
 
+#include "GeneralServer/AuthenticationFeature.h"
```

Comment:
please move includes to the top of the file

Reply:
done